generators/app/templates/Procfile.dev

```diff
@@ -1,2 +1,2 @@
-server: node express.js
-client: rails s -p 3000
+server: cd server && rails s -p 3000
+client: cd client && npm start
```

Comment:
I can't get foreman running in dev, I build a script on `bin` dir. The generator is having some problem to exec the `bin/setup` too.